addons/calendar/models/calendar_alarm.py

```diff
@@ -26,6 +26,7 @@ class Alarm(models.Model):
         compute='_compute_mail_template_id', readonly=False, store=True,
         help="Template used to render mail reminder content.")
     body = fields.Text("Additional Message", help="Additional message that would be sent with the notification for the reminder")
+    notify_responsible = fields.Boolean("Notify Responsible")
```

Comment:
Better to make this field default `False`? Give it a try.

Reply:
> default False

Yes, we can make the field default `False`.

> making it generic is a good idea

Yes, we need to make this field generic. The `sms_notify_responsible` field is defined in the `calendar_sms` module, and we want the same boolean field for WhatsApp. Since SMS and WhatsApp are independent modules, there may be cases where `calendar_sms` is not installed but `whatsapp_calendar` is installed. In such cases, we will not be able to use the field.